### DISCUSSION TITLE
Command + Tab on remote?

### DISCUSSION THREAD

**Discussion by Balky79** [rustdesk/rustdesk#8387] [Q&A]
Hi, 

I'm connecting mac to mac, and it works nice and fast, but looking at the options and settings, I don't see a way to enable/disable/change/pass "command+tab" to remote. 

It quite reduces the regular workflow, once you connect to remote desktop, you expect to work "as on local", or to at least have some sort of "backup" - e.g. fn+cmd+tab ... or something such. 

Is that a thing on RustDesk? Or is this a feature request? 

Thanks!

**Comment by ramarivera** [comment]
Sad to find out this is not supported

**Comment by BasilChessman** [comment]
Are there any plans to make this work?  It's pretty much a deal breaker when keyboard shortcuts don't work.

**Comment by misaelriojasftf** [comment]
no news?
